src/apps/rDSN.monitor/server.py

```diff
@@ -36,16 +39,52 @@ def SendJson(self, r):
         self.response.write(json.dumps(r))
 
     def geneRelate(self,task_code,params):
-        task_list = sorted(ast.literal_eval(Native.dsn_cli_run('query task_list')))
-	call_list = ast.literal_eval(Native.dsn_cli_run('query call '+task_code))
-        caller_list = call_list[0]
-        callee_list = call_list[1]
-	sharer_list = ast.literal_eval(Native.dsn_cli_run('query pool_sharer '+task_code))
+        task_list = sorted(ast.literal_eval(Native.dsn_cli_run('pq task_list')))
+	call_list = ast.literal_eval(Native.dsn_cli_run('pq call '+task_code))
```

Comment:
seems not aligned - not sure this can be correctly run in python; there are a few places with the same issue below.